README.md

```diff
@@ -114,6 +114,12 @@ Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
+
+👤 **Contributer Tatou Roberto**
+
+- GitHub: [@tatouroberto](https://github.com/tatouroberto)
+
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
```

Comment:
- [ ] As requested by the previous reviewer, Kindly consider documenting your project properly with the appropriate information for your users to be able to get enough on the project:
- Prerequisites: knowledge and tools they need to have before they can work or contribute to your project.
- Setup: How they can clone your project onto their local machine.
- Install: What they need to install in order for them to run your project successfully with the commands.
- Usage: How they can run your project on their local machine after cloning it successfully.
- Run tests: How they can run the project's test on their local machine with commands.